src/layouts/SiteLaunchPad/components/SiteLaunchFinalState.tsx

```diff
@@ -72,13 +76,51 @@ const SiteLaunchFailureState = (): JSX.Element => {
   )
 }
 
+const SiteLaunchInProgressState = (): JSX.Element => {
+  return (
+    <>
+      <Text
+        textStyle="h3"
+        textColor="base.content.strong"
+        mt="3rem"
+        mb="1.5rem"
+      >
+        We&apos;re keeping an eye out... Be right back!
+      </Text>
+      <SiteLaunchPendingImage />
+      <Text
+        textStyle="body-1"
+        textColor="base.content.default"
+        mt="1.5rem"
+        textAlign="center"
+      >
+        Come back to this space to track your site status - you may see an error
+        when visiting your domain at this moment. Leave this window open or exit
+        and come back later.
+      </Text>
+      <Button variant="link" colorScheme="neutral" mt="3rem">
+        Back to tasklist
```

Comment:
In the ss, this looks like text more than button, might wanna just check this

Reply:
It's a button with the link variant - this is in line with screen in figma. 
![Screenshot 2023-08-16 at 9 10 42 AM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/e490c694-9032-4711-87e5-c94567b9e7ce)